src/test/sql/sql_translator_test.cpp

```diff
@@ -2492,6 +2492,10 @@ TEST_F(SQLTranslatorTest, CatchInputErrors) {
   EXPECT_THROW(compile_query("WITH q AS (SELECT * FROM int_float), q AS (SELECT b FROM q) SELECT * FROM q;"),
                InvalidInputException);
   EXPECT_THROW(compile_query("COPY no_such_table TO 'a_file.tbl';"), InvalidInputException);
+  EXPECT_THROW(compile_query("SELECT * FROM meta_unknown;"), InvalidInputException);
+  EXPECT_THROW(compile_query("UPDATE meta_unknown SET a = 1;"), InvalidInputException);
+  EXPECT_THROW(compile_query("DELETE FROM meta_unknown;"), InvalidInputException);
+  EXPECT_THROW(compile_query("INSERT INTO meta_unknown (a) VALUES(1);"), InvalidInputException);
```

Comment:
Do we have similar tests for DML on non-modifiable meta tables?

Reply:
This should be covered [here](https://github.com/hyrise/hyrise/blob/b6d92331c43d469eb4f07c3df2beb3db44b87c20/src/test/sql/sql_translator_test.cpp#L1951).